examples/wagtail/core/jinja/home_page.jinja

```diff
@@ -2,16 +2,24 @@
 
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>{{ page.title }}</title>
     <link rel="stylesheet" href="{{ static('css/theme.css') }}">
     <link rel="stylesheet" href="{{ static('css/styles.css') }}">
+    <link rel="stylesheet" href="{{ static('css/ogcio-ds-0.4.4.min.css')}}">
 </head>
-<body>    
+
+<body>
+    <h2>Header built with macros --> Global Component</h2>
```

Comment:
Could change to Header (Global), Header (HTML) titles for example (removes implementation details)